components/org.wso2.carbon.identity.branding.preference.management.core/src/main/java/org/wso2/carbon/identity/branding/preference/management/core/BrandingPreferenceManager.java

```diff
@@ -62,6 +62,17 @@ BrandingPreference getBrandingPreference(String type, String name, String locale
     BrandingPreference resolveBrandingPreference(String type, String name, String locale)
             throws BrandingPreferenceMgtException;
 
+    /**
+     * This API is used to retrieve a resolved branding preference for an application.
+     *
+     * @param name   Name of the application.
+     * @param locale Language preference of the branding.
+     * @return The resolved branding preference. If not exists return the default branding preference.
+     * @throws BrandingPreferenceMgtException if any error occurred.
+     */
+    BrandingPreference resolveApplicationBrandingPreference(String name, String locale)
```

Comment:
make this default

Reply:
Can change the variable name to identifier

Reply:
Addressed with https://github.com/wso2-extensions/identity-branding-preference-management/commit/cf770738c9203eb8c32f84bd8258e30b6acf198a